policies/strategic_planning_committee_draft.md

```diff
@@ -0,0 +1,56 @@
+Are you interested to join Astropy Strategic Planning Committee?
+Here is what to expect.

```

Comment:
Add a section on top about "beta year" to start things out and hammer out what works and does not interatively.